src/itest/java/uk/gov/companieshouse/company/metrics/controller/HealthCheckControllerITest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.company.metrics.controller;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+@SpringBootTest
+public class HealthCheckControllerITest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
```

Comment:
We can also this class, once Cucumber BDD tests introduced we can run integration tests without mocks

Reply:
Removed HealthCheckControllerITest.java